src/symbols/Arrow.tsx

```diff
@@ -112,6 +100,31 @@ export const Arrow: FC<ArrowProps> = ({
 Arrow.defaultProps = {
   size: 1,
   opacity: 0.5,
-  placement: 'end',
   color: '#D8E6EA'
 };
+
+export function getArrowVectors(
```

Comment:
Let's move this to a utils file to keep the components a little cleaner and add some comments.